pallets/creditcoin/src/types.rs

```diff
@@ -61,6 +64,29 @@ impl Blockchain {
 	}
 }
 
+#[derive(Clone, Encode, Decode, Eq, PartialEq, RuntimeDebug, TypeInfo, MaxEncodedLen)]
+pub enum SignatureType {
```

Comment:
This type seems unused - it shouldn't be necessary anyway since it's already covered by `OwnershipProof`